org.lvnauth.LVNAuth.json

```diff
@@ -0,0 +1,110 @@
+{
+    "app-id": "org.lvnauth.LVNAuth",
+    "runtime": "org.freedesktop.Sdk",
+    "runtime-version": "23.08",
+    "sdk": "org.freedesktop.Sdk",
+    "command": "startup.sh",
+    "finish-args": [
+	    "--socket=x11",
+	    "--share=network",
+	    "--share=ipc",
+	    "--socket=pulseaudio",
+	    "--filesystem=home",
+	    "--device=dri"
+        ],
+    "modules": [
+        {
+            "name": "lvnauth",
+            "build-commands": [
+		"install -D flatpak/startup.sh ${FLATPAK_DEST}/bin/startup.sh",
+                "chmod +w ${FLATPAK_DEST}/bin/startup.sh",
+                "cp -r src/* ${FLATPAK_DEST}/bin",
+                "install -D flatpak/org.lvnauth.LVNAuth.desktop /app/share/applications/${FLATPAK_ID}.desktop",
+                "install -D flatpak/lvnauth_16x16.png /app/share/icons/hicolor/16x16/apps/${FLATPAK_ID}.png",
+                "install -D flatpak/lvnauth_32x32.png /app/share/icons/hicolor/32x32/apps/${FLATPAK_ID}.png",
+                "install -D flatpak/lvnauth_64x64.png /app/share/icons/hicolor/64x64/apps/${FLATPAK_ID}.png",
+                "install -D flatpak/lvnauth_128x128.png /app/share/icons/hicolor/128x128/apps/${FLATPAK_ID}.png",
+                "install -D flatpak/lvnauth_256x256.png /app/share/icons/hicolor/256x256/apps/${FLATPAK_ID}.png"
+	    ],
+	    "buildsystem": "simple",
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "https://github.com/jrezai/lvnauth.git",
+                    "commit": "76d832ff64c31fc308b5d296087421a907664387"
+                }
+            ]
+        },
+	"python3-tkinter.json",
+        {
+            "name": "flit-core",
```

Comment:
If you are generating this and below with flatpak-pip-generator it should be in a seperate manifest

Reply:
Which part(s) need to be in a separate manifest?

Reply:
The modules starting from this and below

Reply:
You didn't remove them from the main manifest

Reply:
I forgot to push the change it seems, I just did that, thanks.